com/course/zenkouv/src/Lecture4/task2/util/Path.java

```diff
@@ -0,0 +1,14 @@
+package Lecture4.task2.util;
+
+public class Path {
+
+    public static final String MESSAGE_PATH = "e:\\knight\\message.txt";
```

Comment:
do not  absolute path . this code will work only on your machine 
MESSAGE_PATH = "message.txt" and etc

Reply:
fix